hydroshare/hydroshare
# Resource Landing Page and Discover Linked Functionality

## Code reviewers, I have left explanatory comments in the PR and also commented code

### Testing:

username is czo_sierra and password is in your email, Neal.

1. Fixes https://github.com/hydroshare/hydroshare/issues/4067

- Go to https://www.hydroshare.org observe loading a resource flashes a red dialog box. If things are slow enough or you use your phone to record the screen, you will see it's a ${k} message and an accompanying red warning.
- On http://34.121.245.225:8000/ observe this is no longer the case.

2. Fixes https://github.com/hydroshare/hydroshare/issues/4057

- Go to a resource at http://34.121.245.225:8000/ and click on a keyword.
- Observe you are directed to the Discover page with appropriate exact subject filtering.
- Note checkbox will not display if the number of resources with that keyword do not meet the quantity threshold; not all keyword filters are displayed as checkboxes, as this would take too long to load and consume the entire page.

3. Fixes https://github.com/hydroshare/hydroshare/issues/2855
- Open a resource at http://34.121.245.225:8000/ click on a keyword with a space in it
- Observe resource count in facet
- Click on facet
- Observe resource count
